src/components/search-results/index.js

```diff
@@ -243,7 +243,7 @@ class SearchResults extends Component {
     const gridStyle = searchComplete && searchItems.length === 0 ? hideGridStyle : showGridStyle;
     return (
       <div className='gridContainer'>
-        {!isInitialTag && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
+        {!isInitialTag && ranking && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
```

Comment:
I don't think this is quite valid, as `ranking` will be an empty object (`{}`) by default, which will evaluate to `true` here. We probably need to check that the ranking object has at least one key.

I'd probably look to extract the `noHotelsErrorMessage` out into a function to avoid the conditional getting out of hand as well.